spec/requests/tickets_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe "Tickets"
+  describe "Fetch tickets" do
+    it 'works and return status 201' do
```

Comment:
the name doesn't correspond with https://github.com/KMosc/APITRY/pull/13/files#diff-0d888e45cfd38a87fd4472cb377ce6785b8532af5b65011e6175ecb356025d74R7